config/samples/resources/cloudidsendpoint/cloudids_v1beta1_cloudnetwork.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Google LLC
```

Comment:
Typo? The filename should be `compute_v1beta1_computenetwork.yaml`. The filename matches the kind, and should not mention cloudids.